src/constants.js

```diff
@@ -471,6 +471,11 @@ export const ACTIVITY_FILTER_OPTION_RESOLVED: 'resolved' = 'resolved';
 export const ACTIVITY_FILTER_OPTION_TASKS: 'tasks' = 'tasks';
 export const ACTIVITY_FILTER_OPTION_UNRESOLVED: 'open' = 'open';
 
+/* ------------------ File Activity Action Types ----------- */
+export const ACTION_TYPE_CREATED: 'created' = 'created';
```

Comment:
I'm not sure if there's a concept of "action type" for other features but if there was we could change to `FILE_ACTIVITY_ACTION_TYPE_[...]` to be more specific